app/components/Page/index.js

```diff
@@ -2,34 +2,34 @@ import React from 'react'
 import classNames from 'classnames/bind'
 
 import Header from 'components/Header'
-import styles from './index.css'
+import SimpleHeader from 'components/SimpleHeader'
+import styles from 'assets/css/global.css'
 
 const cx = classNames.bind(styles)
 
+const header = (withoutHeader, simpleHeader, noBorder) => {
+  if (withoutHeader) {
+    return
+  }
+  if (simpleHeader) {
+    return <SimpleHeader noBorder={noBorder} />
+  }
+  return <Header />
+}
+
 const Page = ({
+  page,
+  withoutHeader,
+  simpleHeader,
+  noBorder,
   children,
-  padding,
-  ...props,
-}) => {
-  const classes = cx(
-    styles.container,
-    padding
-  )
-
-  return (
-    <div>
-      {!props.withoutHeader &&
-        <Header
-          account={props.account}
-          logOut={props.logOut}
-          settings={props.settings}
-        />
-      }
-      <div className={classes}>
+}) => (
+    <div className={cx(styles.extension, page)}>
+      <div className={styles.extensionInner}>
+        {header(withoutHeader, simpleHeader, noBorder)}
```

Comment:
What I would do here:
```
{ withHeader && <Header simple={simpleHeader} border={noBorder} />
```

And move the logic of SimpleHeader inside Header component, probably the SimpleHeader is just the header without showing all parts of it, so I would apply the same pattern:
```
const Header = ({...}) => (
    ....
    { fullHeader && ... }
    <div> ... </div>
    { fullHeader && .... }
)
```